### DISCUSSION TITLE
How to get an estimate of sender location using "user_ip"

### DISCUSSION THREAD

**Discussion by NomanKhan3701** [emailjs-com/help#58] [Q&A] [answered]
How do I access the user_ip from an email to get an estimate from where the email was sent. Cuz the user_ip (e.g. 81b037e16aae93d297d05bc0493e29e9) is not in the valid IP address format

**Comment by xr0master** [comment, marked as answer]
The user_ip is the IP address in the md5 hash, as required by the GDPR. However, if there is a strong desire to find the address, then you can find it through brute force. Or check the existing address that you suspect.
